.flake8

```diff
@@ -1,3 +1,4 @@
 [flake8]
 max-line-length = 88
 extend-ignore = E203
+exclude = .git,__pycache__,./pystarport/pystarport
```

Comment:
you excluded the whole pystarport package from python linter?

Reply:
i'll change it, exclude only cosmos, tendermint